### PR TITLE
Fix demo data

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
fixes the demo data loading rake task

#### Why
This has been broken for a while due to validation
changes. Some devs still like to use this.

### CODE REVIEW

File: lib/demo_data/base_claim_generator.rb [hunk 1]
```diff
@@ -112,9 +112,8 @@ def advance_claim_to_state(claim, state)
       begin
         ClaimStateAdvancer.new(claim).advance_to(state)
       rescue => err
-        puts "ERROR: #{err.class} :: #{err.message}"
-        ap claim
-        ap claim.assessment
+        puts "ERROR: #{err.class} :: #{err.message}".red
+        binding.pry
```
Comment (jsugarman) [inline lib/demo_data/base_claim_generator.rb]: remove

File: lib/tasks/claims.rake [hunk 2]
```diff
@@ -55,6 +55,7 @@ namespace :claims do
     params = {num_claims_per_state: 1, num_external_users: 1}.merge(args)
     Rake::Task['claims:demo_data:seed'].invoke
     Rake::Task["claims:demo_data:advocates"].invoke(params[:num_claims_per_state], params[:num_external_users])
+    # Rake::Task["claims:demo_data:advocate_interims"].invoke(params[:num_claims_per_state], params[:num_external_users]) # TODO
```
Comment (jsugarman) [inline lib/tasks/claims.rake:58]: remove or add complete demo data example task

File: lib/demo_data/base_claim_generator.rb (hunk 1 above)
Comment (jsugarman) [inline lib/demo_data/base_claim_generator.rb]: remove

Comment (jsugarman) [inline lib/demo_data/base_claim_generator.rb]: remove

**Review by colinbruce — APPROVED** [review]
:+1:

**Pull request merged**